drivers/nrf_802154_sl_opensource/CMakeLists.txt

```diff
@@ -4,14 +4,16 @@ include(nrf_802154_sl_files.cmake)
 
 if (CONFIG_NRF_802154_SL_OPENSOURCE)
   zephyr_include_directories(${NRF_802154_SL_OPENSOURCE_INCLUDE_DIRS})
-  zephyr_library_sources(
-    ${NRF_802154_SL_OPENSOURCE_SOURCES}
-  )
+  zephyr_library_sources(${NRF_802154_SL_OPENSOURCE_SOURCES})
```

Comment:
It seems this file should be moved out of this repository as well. It includes dependencies to Zephyr